packages/nextjs/pages/_app.tsx

```diff
@@ -18,27 +17,27 @@ import "~~/styles/globals.css";
 const ScaffoldEthApp = ({ Component, pageProps }: AppProps) => {
   const price = useEthPrice();
   const setEthPrice = useAppStore(state => state.setEthPrice);
-  // This variable is required for initial client side rendering of correct theme for RainbowKit
-  const [isDarkTheme, setIsDarkTheme] = useState(true);
-  const { isDarkMode } = useDarkMode();
+  // const isClient = useIsClient();
+  // const isDarkMode = useDarkMode();
 
   useEffect(() => {
     if (price > 0) {
       setEthPrice(price);
     }
   }, [setEthPrice, price]);
 
-  useEffect(() => {
-    setIsDarkTheme(isDarkMode);
-  }, [isDarkMode]);
-
   return (
     <WagmiConfig client={wagmiClient}>
       <NextNProgress />
       <RainbowKitProvider
         chains={appChains.chains}
         avatar={BlockieAvatar}
-        theme={isDarkTheme ? darkTheme() : lightTheme()}
+        // theme={isClient && isDarkMode ? darkTheme() : lightTheme()}
```

Comment:
Have commented out this, this is an alternate solution to https://github.com/scaffold-eth/se-2/pull/299#discussion_r1161985460

With this approach, we don't need an extra state. lol even this can be ignored was just trying to utilize [`useHooks`](https://usehooks-ts.com/) which was already in place and seems clean